packages/manager/modules/vrack/package.json

```diff
@@ -21,38 +21,25 @@
   "files": [
     "dist"
   ],
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.js",
-  "browser": "./dist/umd/vrack.js",
-  "scripts": {
-    "build": "rollup -c --environment BUILD:production",
-    "dev": "rollup -c --environment BUILD:development",
-    "dev:watch": "yarn run dev --watch",
-    "prepare": "yarn run build",
-    "start": "lerna exec --stream --scope='@ovh-ux/manager-vrack' --include-filtered-dependencies -- yarn run build",
-    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-vrack' --include-filtered-dependencies -- yarn run dev",
-    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-vrack' --include-filtered-dependencies -- yarn run dev"
-  },
+  "main": "./src/index.js",
   "dependencies": {
-    "bootstrap4": "twbs/bootstrap#v4.0.0",
     "lodash": "^4.17.15",
     "moment": "^2.22.2"
   },
   "devDependencies": {
     "@ovh-ux/component-rollup-config": "^7.0.0",
-    "bootstrap": "~3.3.7"
+    "bootstrap": "~3.3.7",
+    "bootstrap4": "twbs/bootstrap#v4.0.0"
```

Comment:
Why do you need both ?

Reply:
This has been removed. Thanks for pointing this out. It was not actually required